openreview/venue_request/process/matchingProcess.py

```diff
@@ -0,0 +1,42 @@
+def process(client, note, invitation):
+
+    GROUP_PREFIX = ''
+    SUPPORT_GROUP = GROUP_PREFIX + '/Support'
+    request_form = client.get_note(note.forum)
+    conference = openreview.helpers.get_conference(client, note.forum)
+
+    build_conflicts=note.content.get('build_conflicts')
+
+    matching_group = note.content['matching_group']
+    compute_scores = note.content.get('compute_affinity_scores') == 'Yes'
+    scores = note.content.get('upload_affinity_scores')
+    file_name=None
+
+    if scores:
+        scores_stream = client.get_attachment(id=note.id, field_name='upload_affinity_scores')
```

Comment:
what happens if they says 'Yes' and also pass a file? which process wins? I think the API, right?

Reply:
maybe we can add this check to the preprocess function?

Reply:
Good idea!